force-app/main/default/classes/VariablesDataTypesOperators.cls

```diff
@@ -226,9 +254,9 @@ public with sharing class VariablesDataTypesOperators {
      */
     public static Double complexOrderOfOperations(Integer a, Integer b, Integer c, Integer d, Integer e) {
         // Add parentheses around the multiplication and subtraction operations so that they are performed before division and addition
-        Double result = a * b - c / (Double) d + e;
+        Double result = (((a * b) - c) / (Double) d) + e;
```

Comment:
You don't need the parentheses around a * b because multiplication already comes before subtraction in the order of operations